lfs.h

```diff
@@ -29,8 +29,10 @@ extern "C"
 // Major (top-nibble), incremented on backwards incompatible changes
 // Minor (bottom-nibble), incremented on feature additions
 #define LFS_DISK_VERSION 0x00020001
-#define LFS_DISK_VERSION_MAJOR (0xffff & (LFS_DISK_VERSION >> 16))
-#define LFS_DISK_VERSION_MINOR (0xffff & (LFS_DISK_VERSION >>  0))
+#define LFS_DISK_VERSION_MAJOR_GET(disk_version) (0xffff & (disk_version >> 16))
```

Comment:
No. Do the modification on the Zephyr side.
We try to synchoronize LittleFS from original sources and this is not used anywhere inside LittleFS nor needed for it to operate with Zephyr.